src/core/server/cron/seenCommentsCleanup.ts

```diff
@@ -0,0 +1,74 @@
+import { Config } from "coral-server/config";
+import { MongoContext } from "coral-server/data/context";
+import { retrieveSeenCommentsForDeletion } from "coral-server/models/seenComments/seenComments";
+import { TenantCache } from "coral-server/services/tenant/cache";
+
+import {
+  ScheduledJob,
+  ScheduledJobCommand,
+  ScheduledJobGroup,
+} from "./scheduled";
+
+interface Options {
+  mongo: MongoContext;
+  tenantCache: TenantCache;
+  config: Config;
+}
+
+export const NAME = "Seen Comments Cleanup";
+
+export function registerSeenCommentsCleanup(
+  options: Options
+): ScheduledJobGroup<Options> {
+  const cronTime = options.config.get("seen_comments_cleanup_interval");
+  const job = new ScheduledJob(options, {
+    name: `Daily ${NAME}`,
+    cronTime,
+    command: cleanupSeenComments,
+  });
+
+  return { name: NAME, schedulers: [job] };
+}
+
+const cleanupSeenComments: ScheduledJobCommand<Options> = async ({
```

Comment:
Why not instead utilize a TTL index in MongoDB?

https://docs.mongodb.com/manual/core/index-ttl/

It's designed to do this automatically.